main.go

```diff
@@ -311,6 +314,10 @@ func runPprof(options *Options) {
 
 // createProxyConfig creates proxy.Config from the command line arguments
 func createProxyConfig(options *Options) proxy.Config {
+	if options.HTTPSServerName == "" {
+		options.HTTPSServerName = "AdGuard DNS"
```

Comment:
Let's make it `dnsproxy` by default, makes more sense

```suggestion
		options.HTTPSServerName = "dnsproxy"
```